libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/map/MapViewBinder.kt

```diff
@@ -71,8 +73,15 @@ abstract class MapViewBinder : UIBinder {
         return navigationListOf(
             CameraLayoutObserver(store, mapView, navigationViewBinding),
             LocationComponent(context.locationProvider),
-            reloadOnChange(context.styles.locationPuck) { locationPuck ->
-                LocationPuckComponent(mapView.location, locationPuck, context.locationProvider)
+            reloadOnChange(
+                navigationState,
+                context.styles.locationPuckOptions
+            ) { navState, _ ->
+                locationPuckComponent(
+                    navState,
+                    mapView.location,
+                    context.locationProvider
+                )
```

Comment:
```suggestion
            ) { navState, locationPuckOptions ->
                locationPuckComponent(
                    navState,
                    mapView.location,
                    context.locationProvider,
                    locationPuckOptions,
                )
```
And in `locationPuckComponent` you will be able to use the new argument instead of `context.styles.locationPuckOptions.value`.

Reply:
Done.